### PR TITLE
feat: button feedback for tv users

### PR DESCRIPTION
**Pull request by sebasg0**
Issue:

## What you did
I added user feedback when using tv platforms for react-native-tvos. I will send new prs to improve focus management for tv platforms, so we can test tv components as well.
IMG_4459

## How to test
Use new lib and test feedback out.

- Does this need a new example in examples/native? No
- Does this need an update to the documentation? No

### CODE REVIEW

**Comment by dannyhw** [comment]
@sebasg0 thanks for the contribution!! 🙇 

👀  nice! I didn't know if it would work on tv or not! 😅 

All PR's welcome!

File: app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx [hunk 1]
```diff
@@ -14,6 +14,12 @@ const ButtonText = styled.Text<{ active: boolean }>(({ theme, active }) => ({
   fontSize: 11,
 }));
 
+const styles = StyleSheet.create({
+  focussedButton: {
```
Comment (dannyhw) [inline app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:18]: just a nit pick but I guess it should be focusedButton

File: app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx [hunk 2]
```diff
@@ -14,6 +14,12 @@ const ButtonText = styled.Text<{ active: boolean }>(({ theme, active }) => ({
...
+    backgroundColor: 'grey',
```
Comment (dannyhw) [inline app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:19]: I'm wondering if maybe there is some focus styling we can copy from the web storybook, changing the background is ok but probably there is some nice styling defined already for this. I think the web in some cases uses a blue outline.

Comment (sebasg0) [reply app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:19]: Will this style be ok?
Screen Shot 2022-06-28 at 10 36 53

Comment (sebasg0) [reply app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:19]: got it from web storybook

File: app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx (hunk 1 above)
Comment (sebasg0) [reply app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:18]: Oh sure, I missed that one 😅

File: app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx (hunk 2 above)
Comment (dannyhw) [reply app/react-native/src/preview/components/OnDeviceUI/navigation/Button.tsx:19]: I think that should be fine, although I guess this is the selected style.

In this screenshot you can see the focused item has only the blue line and the text is grey

image

**Pull request closed without merging**